branchpro/apps/_inference.py

```diff
@@ -220,40 +220,74 @@ def update_posterior(self, mean, stdev, tau, central_prob, epsilon=None):
         new_beta = mean / (stdev ** 2)
 
         data = self.session_data.get('data_storage')
-        serial_interval = self.session_data.get(
-            'interval_storage').iloc[:, 0].values
 
         if data is None:
             raise dash.exceptions.PreventUpdate()
 
         time_label, inc_label = data.columns[:2]
 
-        if 'Imported Cases' in data.columns:
-            # Separate data into local and imported cases
-            imported_data = pd.DataFrame({
-                time_label: data[time_label],
-                inc_label: data['Imported Cases']
-            })
+        num_cols = len(self.session_data.get('interval_storage').columns)
```

Comment:
We can save a few lines in this method by defining
```
prior_params = (new_alpha, new_beta)
labels = {'time_key': time_label, 'inc_key': inc_label}
```
and then instantiating the posteriors like this
```
 posterior = bp.LocImpBranchProPosterior(
                    data,
                    imported_data,
                    epsilon,
                    serial_interval,
                    *prior_params,
                    **labels)
```